app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,17 @@
+class AdminMailer < ActionMailer::Base
+  ADMIN_EMAIL_ADDRESS = 'micropurchase@gsa.gov'.freeze
```

Comment:
nice! we should potentially use this constant in some of our other mailers, too, since I know we set this is the `reply-to` for all emails!